spec/hand_rankings/hand_entities/high_card_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'poker_hands/hand_rankings/find_high_card'
+require 'poker_hands/hand_rankings/hand_entities/high_card'
+require 'poker_hands/card'
+require 'spec_helper'
+
+
+RSpec.describe PokerHands::Entities::HighCard do
+  describe '#initialize' do
+    subject { PokerHands::Entities::HighCard.new(cards: hand) }
+
+    context 'HighCard entity' do
+      let(:hand) do
+        [
+          PokerHands::Card.new(4, 'H'),
+          PokerHands::Card.new(11, 'S'),
+          PokerHands::Card.new(8, 'D'),
+          PokerHands::Card.new(2, 'S'),
+          PokerHands::Card.new(9, 'C')
+        ]
+      end
+
+      it 'receives a poker hand' do
+        expect(PokerHands::Entities::HighCard).to receive(:new).with(cards: hand)
+        PokerHands::Entities::HighCard.new(cards: hand)
+      end
+
+      it 'is type \'high card\'' do
+        expect(subject.type).to eq('high card')
+      end
+
+      it 'has strength 1' do
+        expect(subject.strength).to be 1
+      end
+    end
+  end
+
+  describe '#<=>' do
+    context 'hand 1 is the winning hand' do
+      let(:hand_1) do
+        [
+          PokerHands::Card.new(4, 'H'),
+          PokerHands::Card.new(12, 'S'),
+          PokerHands::Card.new(8, 'D'),
+          PokerHands::Card.new(2, 'S'),
+          PokerHands::Card.new(9, 'C')
+        ]
+      end
+      let(:hand_2) do
+        [
+          PokerHands::Card.new(4, 'C'),
+          PokerHands::Card.new(11, 'S'),
+          PokerHands::Card.new(8, 'C'),
+          PokerHands::Card.new(2, 'D'),
+          PokerHands::Card.new(9, 'H')
+        ]
+      end
+      let(:player_1) { PokerHands::FindHighCard.new.call(hand_1) }
```

Comment:
We could consider building our `HighCard` entities directly instead of bringing in another dependency (`FindHighCard`).